ProductsWebApi/ProductsWebApi.Tests/ValidatorTests/UserValidatorTest.cs

```diff
@@ -0,0 +1,52 @@
+﻿using AutoFixture;
+using Microsoft.AspNetCore.Identity;
+using ProductsWebApi.Models;
+using ProductsWebApi.Services;
+using ProductsWebApi.Validator;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using Xunit;
+
+namespace ProductsWebApi.Tests.ValidatorTests
+{
+    public class UserValidatorTest
+    {
+
+        [Theory]
+        [InlineData("", "test")]
+        public void UserValidatorTests_WhenUserNameisEmpty_ShouldThrowValidationException(string userName, string password)
+        {
+            UserValidator userValidator = new UserValidator();
```

Comment:
Please dont repeat this again. 
`var fixture = new Fixture();` in each tests
You already know what to do. 

One of the prior thing to keep in mind is, try avoid making the same PR comments, which you got prior.

Reply:
Fixed this and moved it to the constructor.